packages/terra-table/src/SelectableUtils.js

```diff
@@ -1,7 +1,20 @@
 /**
-* Returns the index of the first selected row for SingleSelectableRows.
-* To be used in the constructor, to set initial state.
-*/
+ * Returns a validated max count for selection based on the rows and max provided.
+ */
+const validatedMaxCount = (rows, maxSelectionCount) => {
```

Comment:
would rename this to be validatedMaxCountSelection to be more representative of what returning

Reply:
Changed to validatedMaxCountSelection